src/adapter.ts

```diff
@@ -22,31 +23,45 @@ import { CasbinRule } from './casbinRule';
 export class SequelizeAdapter implements Adapter {
   private option: SequelizeOptions;
   private sequelize: Sequelize;
+  private hookModelOption: ((option: ModelOptions) => void) | undefined;
 
-  constructor(option: SequelizeOptions) {
+  constructor(option: SequelizeOptions, hookModelOption?: (option: ModelOptions) => void) {
     this.option = option;
+    this.hookModelOption = hookModelOption;
   }
 
   /**
    * newAdapter is the constructor.
    * @param option sequelize connection option
+   * @param hookModelOption hook casbin rule model option
    */
   public static async newAdapter(
-    option: SequelizeOptions
+    option: SequelizeOptions,
+    hookModelOption?: (option: ModelOptions) => void
```

Comment:
Why not use the  `modelOption?: ModelOptions`?

Reply:
Because this option is generated according to the annotation of the model, there are many written values in it, and the user should only modify some of them as needed！

Reply:
I mean `hookModelOption?: (option: ModelOptions) => void` => `modelOption?: ModelOptions`.

Reply:
There are many fields in modeloptions. Generally, users only need to change one or two. If you directly transfer modeloptions, you may need to write a lot of logic for null assignment, and users can't easily get the existing modeloptions

Reply:
@Gabriel-403 It doesn't make sense,  use variables is more intuitive.